dataproc/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 
 [1]: https://pypi.org/project/google-cloud-dataproc/#history
 
+## 0.6.1
+
+11-12-2019 08:24 PST
+
+- docs: add python 2 sunset banner to documentation ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
+
+### Implementation Changes
```

Comment:
After moving the generated entries into the appropriate sections, please remove any empty sections.